src/box/rt_access.c

```diff
@@ -0,0 +1,158 @@
+#include "rt_access.h"
+#include "assoc.h"
+#include "../lua/init.h" /* tarantool_lua_is_builtin_global */
+#include "salad/grp_alloc.h"
+
+static struct mh_strn_strn_t *user_rt_access = NULL;
+static struct mh_strnptr_t *user_rt_universe_access = NULL;
+
+struct mh_strnptr_key_t*
+mh_strnptr_key_alloc(const char *str, uint32_t str_len)
```

Comment:
It seems, it should be declared as `static` (and the same for `mh_strn_strn_key_alloc` below).

Reply:
Done.